redshift-monitoring/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: "{{ .Release.Name }}-cw"
+  namespace: "{{ .Release.Namespace }}"
+  labels:
+{{ include "labels" $ | indent 4 }}
+    {{- range $key, $val := .Values.prometheus.labels }}
+    {{ $key }}: {{ $val | quote }}
+    {{- end}}
+spec:
+  endpoints:
+  - interval: {{ .Values.amazonService.interval }}
+    port: http
+  namespaceSelector:
+    matchNames:
+    - "{{ .Release.Namespace }}"
+  selector:
+    matchLabels:
```

Comment:
These might need updating, depending on which labels the `prometheus-cloudwatch-exporter` chart creates.

Reply:
Ideally it already follows the new convention

```
      app.kubernetes.io/name: prometheus-cloudwatch-exporter
      app.kubernetes.io/instance: {{ .Release.Name }}
```